server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -299,10 +299,11 @@ public final class IndexSettings {
         Property.Deprecated
     );
     public static final TimeValue DEFAULT_REFRESH_INTERVAL = new TimeValue(1, TimeUnit.SECONDS);
+    public static final TimeValue MINIMUM_REFRESH_INTERVAL = new TimeValue(-1, TimeUnit.MILLISECONDS);
     public static final Setting<TimeValue> INDEX_REFRESH_INTERVAL_SETTING = Setting.timeSetting(
         "index.refresh_interval",
         DEFAULT_REFRESH_INTERVAL,
```

Comment:
Shouldn't this value be cluster default value? If we create an index without explicit `refresh_interval` and cluster default is `10s`, assuming we are getting constant search traffic (100 per seconds), how frequently a refresh happen?

Reply:
Got it. I did not notice changes to `AsyncRefreshTask` earlier.